openstack/barbican/templates/etc/_secrets.conf.tpl

```diff
@@ -1,17 +1,16 @@
 [DEFAULT]
-sql_connection = {{ include "db_url_mysql" . }}
 
 {{ include "ini_sections.default_transport_url" . }}
 
+[database]
+sql_connection = {{ include "db_url_mysql" . }}
```

Comment:
According to the release notes, the key should be `connection`, not `sql_connection`